### PR TITLE
DEV-2403 Fix creating MH sidecar

### PR DESCRIPTION
**Pull request by spacid**
A MediaHaven sidecar will always contain a Dynamic tag including a PID.

There was some edge case in which the dynamic tag was not yet created when adding the PID. As a fix, explicitly create the dynamic tag upfront.

### CODE REVIEW

**Comment by spacid** [comment]
To add: test

File: tests/helpers/test_xml.py [hunk 1]
```diff
@@ -49,6 +56,13 @@ def minicar_xml():
     return xml
 
 
+@pytest.fixture
+def mh_sidecar_material_artwork_nimimal_rep_xml():
```
Comment (RudolfDG) [inline tests/helpers/test_xml.py:60]: typo ;)
